src/services/benefit/benefits.tsx

```
@@ -289,3 +289,21 @@ export const fetchVCJson = async (url: string) => {
+		const token = localStorage.getItem('authToken');
+		const response = await axios.get(
+			`${apiBaseUrl}/content/eligibility-check/${id}`,
+
+			{
+				headers: {
+					Authorization: `Bearer ${token}`,
+				},
+			}
+		);
+		return response.data;
+	} catch (error: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)